Marlin/gcode.h

```diff
@@ -139,6 +149,17 @@ class GCodeParser {
       return b;
     }
 
+    static volatile bool seen_any() {
+      return strlen(command_args)!=0; // todo: is this right?
```

Comment:
Yeah, that makes good sense -- I just didn't know how command_args was left, so wasn't sure of the best way to detect 'empty', hence the 'is this right?' ...

Thanks!